dsl/kotless/kotless-lang/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ version = rootProject.version
 
 dependencies {
     api(project(":dsl:common:lang-common"))
+    implementation("com.microsoft.azure.functions", "azure-functions-java-library", "1.2.2")
```

Comment:
Not sure that the best way is to bundle Azure right into dsl artifact... Maybe we should split them into `aws` and `azure` part?

Reply:
I tried to split these parts but looks like some cloud-specific libraries still include into artifacts